src/middlewared/middlewared/api/base/types/base/path.py

```diff
@@ -0,0 +1,42 @@
+from pathlib import Path
```

Comment:
This entire file can be removed and replaced with the built-in `FilePath` builtin pydantic annotation. Please use that instead.